org.eclipse.shellwax.target/target-platform.target

```diff
@@ -8,6 +9,10 @@
 	<unit id="org.eclipse.ui.genericeditor" version="0.0.0"/>
 	<unit id="org.eclipse.lsp4e" version="0.0.0"/>
 </location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+	<repository location="https://download.eclipse.org/wildwebdeveloper/snapshots/"/> 
+	<unit id="org.eclipse.wildwebdeveloper.embedder.node" version="0.0.0"/>
```

Comment:
org.eclipse.wildwebdeveloper.embedder.node.feature.feature.group is the unit to be included as it will ensure that the respective arch specific fragment is available in the TP. In your way only the noarch part (which simply redirects to PATH) is available in the TP.

Reply:
the included unit is change to be: `org.eclipse.wildwebdeveloper.embedder.node.feature.feature.group`